### ISSUE TITLE
when use moveHtmlContainer tooltip not correct show

### ISSUE THREAD

**Issue by HASSANDL** [amcharts/amcharts4#2659]
when use moveHtmlContainer tooltip not correct show
```
class AMCharts extends PureComponent {
    constructor(props) {
        super(props);

        const options = {...};

        // create chart
        this.chart = am4core.createFromConfig(options, undefined, options.type);
    }

    componentDidMount() {
        this.chart.moveHtmlContainer(this.chartId);
    }
}
```

![image](https://user-images.githubusercontent.com/22093903/86439490-e57b8480-bd1d-11ea-9667-a655e8411c7b.png)

```
class AMCharts extends PureComponent {
    constructor(props) {
        super(props);

        this.options = {...};
    }

    componentDidMount() {
        // create chart
        this.chart = am4core.createFromConfig(this.options, this.chartId, this.options.type);
    }
}
```
![image](https://user-images.githubusercontent.com/22093903/86439576-0f34ab80-bd1e-11ea-9405-34115d455ef1.png)

**Comment by martynasma** [comment]
Thanks for reporting this. We're looking into it.

Repro case:
https://codepen.io/team/amcharts/pen/ddd1bdc003cb149d0300064a16954bdd?editors=0010

**Comment by martynasma** [comment]
Fixed in [4.9.30](https://github.com/amcharts/amcharts4/releases/tag/4.9.30).

## [4.9.30] - 2020-07-10

### Added
- `am4core.options.suppressWarnings` global option added (default: `false`). If set to `true`, the charts won't output any warnings into console (e.g. "Chart was not disposed").

### Fixed
- Drill-down map with `geodataSource` problem fixed (since last release only).
- Issues with cursor/tooltip and other element positioning fixed, when chart was being initialized outside of DOM (no container on init).

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
